{"uid":"213d4521a8aff4de","name":"test_change_order_status[canceled-api-18]","fullName":"tests.endpoints.test_orders#test_change_order_status","historyId":"64a9041c793eaa2bb9f015ab6b096f49","time":{"start":1733146278434,"stop":1733146279457,"duration":1023},"status":"broken","statusMessage":"pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nusers\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing\nmeta\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing","statusTrace":"api_client = <api_handler.api_client.ApiClient object at 0x7fd731785370>\nstatus = 'canceled', x_task_value = 'api-18'\n\n    @allure.feature(\"Orders API\")\n    @allure.story(\"Change order status\")\n    @pytest.mark.parametrize(\"status, x_task_value\", [(\"canceled\", \"api-18\")])\n    def test_change_order_status(api_client, status, x_task_value):\n        users_response = api_client.get_users(x_task_value, 0)\n>       users_dto = Users(**users_response.json())\nE       pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nE       users\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\nE       meta\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\n\ntests/endpoints/test_orders.py:60: ValidationError","flaky":true,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1733146274602,"stop":1733146274603,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup_test","time":{"start":1733146274604,"stop":1733146276211,"duration":1607},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"worker_id","time":{"start":1733146274604,"stop":1733146274604,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"api_client","time":{"start":1733146274604,"stop":1733146274604,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1733146274604,"stop":1733146274604,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nusers\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing\nmeta\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing","statusTrace":"api_client = <api_handler.api_client.ApiClient object at 0x7fd731785370>\nstatus = 'canceled', x_task_value = 'api-18'\n\n    @allure.feature(\"Orders API\")\n    @allure.story(\"Change order status\")\n    @pytest.mark.parametrize(\"status, x_task_value\", [(\"canceled\", \"api-18\")])\n    def test_change_order_status(api_client, status, x_task_value):\n        users_response = api_client.get_users(x_task_value, 0)\n>       users_dto = Users(**users_response.json())\nE       pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nE       users\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\nE       meta\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\n\ntests/endpoints/test_orders.py:60: ValidationError","steps":[],"attachments":[{"uid":"1f85a4185951a9b0","name":"log","source":"1f85a4185951a9b0.txt","type":"text/plain","size":320},{"uid":"62bf718df10eefba","name":"stderr","source":"62bf718df10eefba.txt","type":"text/plain","size":314}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"setup_test::0","time":{"start":1733146280827,"stop":1733146280827,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"feature","value":"Orders API"},{"name":"story","value":"Change order status"},{"name":"parentSuite","value":"tests.endpoints"},{"name":"suite","value":"test_orders"},{"name":"host","value":"fv-az888-715"},{"name":"thread","value":"2802-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.endpoints.test_orders"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"status","value":"'canceled'"},{"name":"x_task_value","value":"'api-18'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":3,"broken":2,"skipped":0,"passed":3,"unknown":0,"total":8},"items":[{"uid":"c6ba01c3ea536815","reportUrl":"https://ils-808.github.io/qahack/100//#testresult/c6ba01c3ea536815","status":"failed","statusDetails":"AssertionError: Failed to change order status 3f2d5e6c-a707-445a-9993-38086cec4e6f\nassert 422 == 200\n +  where 422 = <Response [422]>.status_code","time":{"start":1733146065555,"stop":1733146069823,"duration":4268}},{"uid":"b04a62a3c1d1498c","reportUrl":"https://ils-808.github.io/qahack/99//#testresult/b04a62a3c1d1498c","status":"broken","statusDetails":"pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nusers\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing\nmeta\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing","time":{"start":1733145945309,"stop":1733145946174,"duration":865}},{"uid":"636a801b5f0e0ba1","reportUrl":"https://ils-808.github.io/qahack/98//#testresult/636a801b5f0e0ba1","status":"passed","time":{"start":1733145784776,"stop":1733145788378,"duration":3602}},{"uid":"fa9a6b862b5c5133","reportUrl":"https://ils-808.github.io/qahack/63//#testresult/fa9a6b862b5c5133","status":"passed","time":{"start":1733086112052,"stop":1733086115590,"duration":3538}},{"uid":"a5aa9dc8c20c7654","reportUrl":"https://ils-808.github.io/qahack/62//#testresult/a5aa9dc8c20c7654","status":"failed","statusDetails":"AssertionError: Failed to change order status 816f2b3e-c086-4265-8b64-41a33b700752\nassert 422 == 200\n +  where 422 = <Response [422]>.status_code","time":{"start":1733084992322,"stop":1733084995794,"duration":3472}},{"uid":"ddb0646bc843cebe","reportUrl":"https://ils-808.github.io/qahack/61//#testresult/ddb0646bc843cebe","status":"failed","statusDetails":"AssertionError: Failed to change order status dabda9a8-a8ff-421c-9768-297b96b4d1e7\nassert 422 == 200\n +  where 422 = <Response [422]>.status_code","time":{"start":1733084770789,"stop":1733084774280,"duration":3491}},{"uid":"535d4b75eed55765","reportUrl":"https://ils-808.github.io/qahack/60//#testresult/535d4b75eed55765","status":"passed","time":{"start":1733084491012,"stop":1733084496204,"duration":5192}}]},"tags":[]},"source":"213d4521a8aff4de.json","parameterValues":["'canceled'","'api-18'"]}